subjects/java/checkpoints/singleton-blueprint/README.md

```diff
@@ -0,0 +1,41 @@
+## Singleton Blueprint
+
+### Instructions
+
+You are given an incomplete singleton class. Complete the class to demonstrate your understanding of how the Singleton design pattern works. The Singleton pattern ensures that a class has only one instance and provides a global point of access to that instance.
```

Comment:
The first Singleton appearance is missing a capital S like the others, and I Would highlight all the `Singleton` words with the backticks.

Reply:
done